source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -75,10 +75,10 @@ export class PropertySheetCategory {
                 let header = new Adaptive.TextBlock();
```

Comment:
Could we test this out in the actual website designer as well while you are at it? Thanks!

Reply:
It is the exact same thing. Also the designer is used in third party applications I can't test - if I can't rely on testing the designer component on its own (but I can), I think we have a problem.